libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx

```diff
@@ -19,10 +18,16 @@ export const GenericFormCancelButton = ({
   return (
     <StyledGenericFormCancelButton>
       <Button
+        sx={{
+          background: '#ffffff',
```

Comment:
Please, don't use hard-coded colors, where possible. 
This color is existing in `variables` so you can use it from there

Reply:
sorry, will replace this with variable as well.